src/server/pegasus_server_impl_init.cpp

```diff
@@ -81,6 +81,20 @@ DSN_DEFINE_validator(read_amp_bytes_per_bit, [](const int64_t read_amp_bytes_per
             (read_amp_bytes_per_bit & (read_amp_bytes_per_bit - 1)) == 0);
 });
 
+DSN_DEFINE_uint64("pegasus.server",
+                  rocksdb_abnormal_batch_get_size_threshold,
```

Comment:
`batch_get_size` => `batch_get_bytes`,  explain units will be clearer

Reply:
ok